apps/pie-microsite/src/assets/styles/_variables.scss

```diff
@@ -1 +1,2 @@
-$content-max-width: 80ch;
+$content-max-width: 65ch;
+$wide-screen-content: 1200px;
```

Comment:
I think we could perhaps rename this to something a little more descriptive. Perhaps `$wide-screen-content-max-width`? Or something else :)